packages/js/src/ui/components/primitives/Tabs/TabsPanel.tsx

```diff
@@ -0,0 +1,24 @@
+import { ParentProps } from 'solid-js';
+import { useStyle } from '../../../helpers';
+import { useTabsContext } from './Tabs';
+
+type TabsPanelProps = ParentProps & {
+  value: string;
+  class?: string;
+};
+
+export const TabsPanel = (props: TabsPanelProps) => {
+  const style = useStyle();
+  const { activeTab } = useTabsContext();
+
+  return (
+    <div
+      class={style('tabsPanel', `${activeTab() === props.value ? 'nt-block' : 'nt-hidden'} ${props.class ?? ''}`)}
```

Comment:
☑️Please make use of the `cn` helper here to make this more readable.
https://www.npmjs.com/package/clsx